### DISCUSSION TITLE
Agente Glpi da erro ao tentar se comunicar com o servidor glpi

### DISCUSSION THREAD

**Discussion by Edson-cmyk** [glpi-project/glpi-agent#721] [Q&A]
Olá, estou tentando utilizar o glpi agent em um parque de maquinas mais no log do Glpi agent é mostrado esses dois erros ao tentar mandar o inventario da maquina para o servidor Glpi, não consegui encontrar uma solução para esse erro:

```
1 - [Wed Jul 24 14:24:30 2024][error] [http client] unexpected content, starting with: <html><body><script type="text/javascript" src="/aes.js" ></script><script>function toNumbers(d){var e=[];d.replace(/(..

2 - [Wed Jul 24 14:24:30 2024][error] No supported answer from server at http://hostname.msahosting.net/marketplace/glpiinventory/

```

**Comment by g-bougard** [comment]
Hello @Edson-cmyk 

I moved your issue in Q&A discussions as this is a better place for it. Anyway in issues, we don't accept issues written in another language than english.

On my side, I won't try to translate it and I just understand you have a problem of communication between agent and your server. The returned error is somewhat clear to me: some html code is inserted in front of the server answer. Maybe you have a wrong plugin, so try to disable them to see if one is inserting this code in the wrong way. Then if this works, enable them one by one until you find which one is working bad and forget to use it and also report the problem to its maintainers. If this is not a plugin, the problem is still server side so investigate your server configuration and web server setup.
